README.md

```diff
@@ -2,6 +2,8 @@
 
 Welcome to the Guardian CDK library! This library contains a number of reusable patterns and constructs which can be used to build up your AWS Cloudformation stacks.
 
+📖 API documentation can be found [here](https://guardian.github.io/cdk/).
```

Comment:
Only a little thing but I think it's preferable to link from some more descriptive text. Admittedly, I'm struggling to reword this sentence but maybe something like:

```suggestion
📖 View the available components in the [API documentation](https://guardian.github.io/cdk/).
```